README.md

```diff
@@ -72,17 +71,18 @@ The section below contains the APIs we're planning to implement for our 1.0 rele
 
 | API Method                                    | Native Equivalent (Android/iOS)                              |
 |-----------------------------------------------|--------------------------------------------------------------|
-| `Instabug.logDebug(String message)            | `InstabugLog.d(String message)`<br>`+ [IBGLog logDebug:]`    |
-| `Instabug.logVerbose(String message)`         | `InstabugLog.v(String message)`<br>`+ [IBGLog logVerbose:]`  |
-| `Instabug.logInfo(String message)`            | `InstabugLog.i(String message)`<br>`+ [IBGLog logInfo:]`     |
-| `Instabug.logWarn(String message)`            | `InstabugLog.w(String message)`<br>`+ [IBGLog logWarn:]`     |
-| `Instabug.logError(String message)`           | `InstabugLog.e(String message)`<br>`+ [IBGLog logError:]`    |
+| `logDebug(String message)                     | `d(String message)`<br>`+ logDebug:`                         |
```

Comment:
Still missing an ` 

`logDebug(String message)`

Reply:
🤦‍♂️ we need a markdown linter.